src/app/chats/[chatId]/tarot-reading/[resultId]/components/TarotInteraction.tsx

```diff
@@ -0,0 +1,165 @@
+"use client";
+
+import * as motion from "motion/react-client";
+import Image from "next/image";
+import catHand from "@/shared/assets/images/catHand.png";
+import cardBack from "@/shared/assets/images/cardBack.jpg";
+import cardFront from "@/shared/assets/images/Card1.jpg";
+import styled from "styled-components";
+import { easeInOut } from "motion";
+import { SetStateAction, useState, Dispatch } from "react";
+
+type PropTypes = {
+  setTarotInteractation: Dispatch<SetStateAction<boolean>>;
+};
+
+const putCardtransition = {
+  duration: 0.6,
+  ease: easeInOut,
+};
+
+const handTransition = {
+  duration: 0.6,
+  delay: 0.6,
+  ease: easeInOut,
+};
+
+const cardRotateTransition = {
+  delay: 1.2,
+  duration: 0.6,
+};
+
+const TarotInteraction = ({ setTarotInteractation }: PropTypes) => {
+  const [isVisible, setIsVisible] = useState(true);
+
+  const routingTarotResult = () => {
+    setTimeout(() => {
+      setTarotInteractation(false);
+    }, 1000);
+  };
+  return (
+    <TarotWaitingWrapper>
+      <Center>
+        <Title>
+          두근두근 <br />
+          어떤 카드가 나올지 기대된다냥
+        </Title>
+        <TarotAnimationBackground>
+          <TaroAnimationWrapper
+            initial={{ opacity: 0, y: 200 }}
+            animate={{ opacity: 1, y: 0 }}
+            transition={putCardtransition}
+          >
+            <CardInner
+              animate={{ rotateY: -180 }}
+              transition={cardRotateTransition}
+              onAnimationComplete={routingTarotResult}
+            >
+              <CardFront alt="cardFront" src={cardFront}></CardFront>
```

Comment:
img는 self closing 태그이므로 다음과 같이 작성해도 괜찮을 것 같아요!
```suggestion
              <CardFront alt="cardFront" src={cardFront} />
```

Reply:
동의합니다 이부분 이전 프로젝트에서 자동화 해두어서 신경 못썼는데 eslint에 아래 설정을 하면서 자동화 해두었어요! 
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/self-closing-comp.md